test/spec/testData.spec.js

```diff
@@ -411,6 +411,157 @@ angular.module('confRegistrationWebApp').service('testData', function () {
           },
         ],
       },
+      {
```

Comment:
@canac Do you think I need to add `conference` to line 9 on [pageRegistration.html](https://github.com/CruGlobal/conf-registration-web/blob/44447f09d06ece3cceaccf3e00bac31c2e92adff/app/views/components/pageRegistration.html) ng-`show="blockVisible(block)"`

Reply:
I don't think so. I'm not fully sure how, but it appears that `pageRegistration.html` is connected to the `blockRegistration` controller, which sets up `$scope.blockVisible`. That's the function that the template is calling.